pymc3/distributions/transforms.py

```diff
@@ -105,12 +105,31 @@ def jacobian_det(self, x):
 log = Log()
 
 
+class Logexpm1(ElemwiseTransform):
```

Comment:
super nitpick, not very important, but easier to change now: `LogExpM1`, and `log_exp_m1`

Reply:
So the class name `LogExpM1` and `name = "log_exp_m1"`?

Reply:
exactly -- convention is underscore to separate words in snake case, capital letter in CamelCase (again, this isn't important, but might give some clues to users figuring out how to parse the word :D)